Library/Homebrew/cmd/update-report.rb

```diff
@@ -22,18 +22,22 @@ def update_report
     HOMEBREW_REPOSITORY.cd do
       analytics_message_displayed =
         Utils.popen_read("git", "config", "--local", "--get", "homebrew.analyticsmessage").chuzzle
+      caskanalyticsmessage =
+        Utils.popen_read("git", "config", "--local", "--get", "homebrew.caskanalyticsmessage").chuzzle
       analytics_disabled =
         Utils.popen_read("git", "config", "--local", "--get", "homebrew.analyticsdisabled").chuzzle
       if analytics_message_displayed != "true" &&
+         caskanalyticsmessage != "true" &&
          analytics_disabled != "true" &&
          !ENV["HOMEBREW_NO_ANALYTICS"] &&
          !ENV["HOMEBREW_NO_ANALYTICS_MESSAGE_OUTPUT"]
+
         ENV["HOMEBREW_NO_ANALYTICS_THIS_RUN"] = "1"
         # Use the shell's audible bell.
         print "\a"
 
         # Use an extra newline and bold to avoid this being missed.
-        ohai "Homebrew has enabled anonymous aggregate user behaviour analytics."
+        ohai "Homebrew (and Cask) have enabled anonymous aggregate user behaviour analytics."
```

Comment:
Should this stay “has” since “and Cask” is in parentheses?

Reply:
This feels weird anyways since everything is in the Homebrew org. Maybe “Homebrew has enabled anonymous aggregate user behaviour analytics for formulae and casks.”?

Reply:
That's exactly why `ohai` shouldn't truncate text.

Reply:
@reitermarkus That's unrelated to `ohai`; even a `puts` that's wrapped by the terminal at >80 characters is uglier than one that's been done by humans (e.g. is it wrapped in the middle of a word or not),